toolchain/lower/file_context.h

```diff
@@ -28,6 +31,13 @@ class FileContext {
     int32_t column_number;
   };
 
+  // Describes a specific function's body fingerprint
+  struct SpecificFunctionFingerprint {
+    llvm::BLAKE3Result<32UL> function_common_fingerprint;
+    llvm::BLAKE3Result<32UL> function_specific_fingerprint;
+    llvm::SmallVector<SemIR::SpecificId> calls;
```

Comment:
It might be nice to have comments explaining the differences between these here, or pointing to where I should look to understand this.